src/main/java/com/example/restapi/controller/EmployeeController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.restapi.controller;
+
+import com.example.restapi.entities.Employee;
+import com.example.restapi.services.EmployeeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/v1")
```

Comment:
this should be "/employees" instead of "/api/v1", don't always use version v1 when you won't know if it has different versions.
The resource name should be plural names - checkout this Resource Naming Convention https://restfulapi.net/resource-naming/

Reply:
Fixed